beprof/curve.py

```diff
@@ -68,10 +73,44 @@ def y_at_x(self, x):
 
     def __str__(self):
         ret = "shape: {}".format(self.shape) + \
-              " X : [{:4.3f},{:4.3f}]".format(min(self.x), max(self.x)) + \
-              " Y : [{:4.6f},{:4.6f}]".format(min(self.y), max(self.y))
+              "\nX : [{:4.3f},{:4.3f}]".format(min(self.x), max(self.x)) + \
+              "\nY : [{:4.6f},{:4.6f}]".format(min(self.y), max(self.y)) + \
+              "\nMetadata : " + str(self.metadata)
         return ret
 
+    # functions not ready yet, added for testing metadata purpose
```

Comment:
Remove it from this PR. It doesn't have anything in common with metadata handling.